### PR TITLE
feat: make catalogue fix to content left

### PR DESCRIPTION
**Pull request by pftom**

### CODE REVIEW

File: src/components/SideBarLeft.tsx [hunk 1]
```diff
@@ -5,10 +5,11 @@ import StepList, { StepListProps } from './StepList';
 
 /* tslint:disable-next-line */
 const SideBarRightWrapper = styled.div`
```
Comment (mRcfps) [inline src/components/SideBarLeft.tsx]: Is this `SideBarLeftWrapper`?

File: src/components/SideBarLeft.tsx [hunk 2]
```diff
@@ -5,10 +5,11 @@ import StepList, { StepListProps } from './StepList';
...
-  width: 240px;
+  width: 220px;
   background-color: rgba(216, 216, 216, 0.1);
   height: 637px;
   margin-top: 90px;
```
Comment (mRcfps) [inline src/components/SideBarLeft.tsx]: I think this `margin-top` is too big, making the sidebar a little bit low.

**Comment by mRcfps** [comment]
Besides, when there are too many steps, can the sidebar scroll?

**Comment by pftom** [comment]
@mRcfps  Yes, this problem has been solved by #67 . 
And now has following two features:

- There is no scrollbar when child content height is overflow parent height.
- And when child content height is overflow parent height, the scrollbar show up.

File: src/components/SideBarLeft.tsx (hunk 1 above)
Comment (pftom) [reply src/components/SideBarLeft.tsx]: Yeah, thx! I'll fix this right now.

File: src/components/SideBarLeft.tsx (hunk 2 above)
Comment (pftom) [reply src/components/SideBarLeft.tsx]: Ok, I'll try another series of `margin-top`.

**Pull request merged**